app/frontend/src/components/forms/submission/MySubmissionsTable.vue

```diff
@@ -119,11 +126,20 @@ export default {
           sortable: false,
         },
       ];
-      if (this.showStatus || !this.formId) {
-        headers.splice(3, 0, {
-          text: 'Completed Date',
+      // Uncomment if you want to add the Completed Date column back to the list
+      // if (this.showStatus || !this.formId) {
+      //   headers.splice(3, 0, {
+      //     text: 'Completed Date',
+      //     align: 'start',
+      //     value: 'completedDate',
+      //   });
+      // }
```

Comment:
Can remove comment if we're taking completed date out of requirements

Reply:
removed the unnecessary comments in new changes